documentation/books/evaluating/master-docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Using {ProductLongName}</title>
```

Comment:
We have similar file for the other docs. But do we actually need these here? We do not use `ccutil` in Strimzi.

Reply:
I use it for locals builds, so I propose we leave it for now.

Reply:
I agree as well, we should remove this stuff here.